docs/decisions/0013-microsoft-fhir-converter.md

```diff
@@ -0,0 +1,105 @@
+
+## Decision Drivers
+
+**Robustness** - The convert must be able to reliably and accurately convert data to FHIR. 
+
+**Hl7v2 Support** - The converter must support the conversion of Hl7v2 to FHIR. 
+
+**C-CDA Support** - The converter must support conversion of C-CDA to FHIR.
+
+**Platform Agnostic** The converter should not depend on any functionality specific to an infrastructure provider.
+
+**Ease of Implementation** - Ideally the converter will be easy implement in a variety of locations.
+
+
+## Considered Options
+
+### Microsoft FHIR Converter CLI
+
+The [Microsoft FHIR Converter](https://github.com/microsoft/FHIR-Converter) is the conversion utility behind the `$convert-data` endpoint on the Microsoft FHIR server and also offers a CLI implementation.
+
+Pros:
+- The same conversion tool we are already familiar with
+- Support for both Hl7v2 and C-CDA
+- Can be installed on Windows, MacOS, and Linux systems.
+- Open source and actively maintained by Microsoft.
+
+Cons:
+- Installation requires building from source.
+- Written in C# with .NET framework making integration with PHDI's primarily Python based work challenging.
+
+### LinuxForHealth FHIR Converter
+
+The [LinuxForHealth FHIR Converter](https://github.com/LinuxForHealth/hl7v2-fhir-converter) is an open source Hl7v2 to FHIR conversion utility written in Java.
+
+Pros:
+- Supports Hl7v2
+- Open source 
+- Can be built on any system with a JVM.
+
+Cons:
+- Does not support C-CDA
+- Installation process is non-trivial
+
+### Report Stream Fhirengine
+
+The Report Stream project is developing a FHIR conversion tool called [fhirengine](https://github.com/CDCgov/prime-reportstream/tree/master/prime-router/src/main/kotlin/fhirengine). It is open source, written in Kotlin, and supports bi-directional Hl7v2 and FHIR conversion.
+
+Pros:
+- Supports Hl7v2
+- Supports conversion from FHIR back to Hl7v2.
+- Can be built on any system with a JVM.
+- Designed, built, and maintained by another project with in PRIME.
+
+Cons:
+- Does not support C-CDA.
+- Installation process is non-trivial.
+ 
+
+### CDC eCR Team FHIR Converter
+
+The Electronic Case Reporting team at CDC is developing a FHIR conversion tool.
+
+Pros:
+- Supports C-CDA
+- Developed by another team at CDC.
+
+Cons:
+- We believe that it does not support Hl7v2, but have not be able to confirm this.
+- The eCR team has not be responsive to setting up a meeting to dsicuss their converter.
+
+### GCP Healthcare Data Harmonization
```

Comment:
We should be more clear in our description of this option. Based on a brief read of the documentation, the tool appears to be a general purpose converter from one data format to another. Some cons that jump out to me are that there don't appear to be any pre-built templates, meaning we'd either have to create them ourselves or have the STLTs do it, and running the code outside of Google's cloud environment seems to require having Go installed. You've mentioned some of this below, but I think we can make it a little more explicit.